examples/erc20/lib.rs

```diff
@@ -55,24 +60,37 @@ mod erc20 {
     /// The ERC-20 result type.
     pub type Result<T> = core::result::Result<T, Error>;
 
+    impl SpreadAllocate for Erc20 {
```

Comment:
Should we add a note here that this is intended to only be temporary and will (hopefully) be replaced by #995?

Reply:
I think it is better to merge this PR after merging the one PR that improves the SpreadLayout traits.